### PR TITLE
Ran pre-commit, added licenses

### PR DESCRIPTION
**Pull request by dherrada**
Not sure what to do about adafruit_minimqtt/matcher.py since it uses a different license

### CODE REVIEW

**Comment by kattni** [comment]
@dherrada This one failed on pre-commit license issues. Please take a look. Ping me when you're ready for a review.

**Review by kattni — APPROVED** [review]

**Comment by brentru** [comment]
@dherrada  the SPDX for the EPL license looks OK to me.

File: adafruit_minimqtt/adafruit_minimqtt.py [hunk 1]
```diff
@@ -1,28 +1,11 @@
-# The MIT License (MIT)
-#
-# Copyright (c) 2019 Brent Rubell for Adafruit Industries
+# SPDX-FileCopyrightText: 2019 Brent Rubell for Adafruit Industries
 #
+# SPDX-License-Identifier: MIT
+
 # Original Work Copyright (c) 2016 Paul Sokolovsky, uMQTT
```
Comment (brentru) [inline adafruit_minimqtt/adafruit_minimqtt.py:5]: @dherrada do these other copyrights need to be added to the `SPDX-FileCopyrightText`?

Comment (dherrada) [reply adafruit_minimqtt/adafruit_minimqtt.py:5]: @brentru I'm not actually sure. I think they might.

**Pull request merged**